### PR TITLE
Update HCT priority queue with Y2 criteria

### PR DESCRIPTION
**Pull request by kschwabefry**
Commits Included:
- HCT ETL: add daily_travel question, update study start date 
- shipping.__uw_priority_queue_v1: Update for HCT Y2 logic
- offer-uw-testing: Add travel test reason, update Y2 study start date

Testing: I've tested the database changes locally and it seems ok, we'll have Jess monitor for a few days by comparing the view and her invitation list to make sure its what she expects. I haven't been able to test out the ETL change yet locally, because of errors with the cachetool dependency - still trying to tinker with my locally installed versions of that to get it working again, but wanted to get this PR out for some reviews sooner rather than later.

### CODE REVIEW

**Comment by davereinhart** [comment]
@kschwabefry ETL changes look good. I was able to test that part and confirm the `daily_travel` values are populating in my local DB. I'm not getting any rows back from `shipping.__uw_priority_queue_v1` view though, even after refreshing the materialized views. I need to dig into that query a little more to see why.

**Comment by kschwabefry** [comment]
> @kschwabefry ETL changes look good. I was able to test that part and confirm the `daily_travel` values are populating in my local DB. I'm not getting any rows back from `shipping.__uw_priority_queue_v1` view though, even after refreshing the materialized views. I need to dig into that query a little more to see why.

Wonderful, thanks for testing that, glad to hear that's populating. I have the sqitch changes deployed locally and dumped the prod data yesterday, and `shipping.__uw_priority_queue_v1` is currently pulling up ~139 records in my local db, with symptomatic/exposure as reason selected for testing. The exact number of results in that view would be dependent on that state of the data when you dump from production though, and when you run the query. Not sure if this is the issue, but if you didn't have any recent daily attestation encounters in your local db, that query might not have results, because it is looking for positive attestations within a recent timeframe. I'd expect if you just ingested recent travel attestations, that those would show up in the view though.

**Comment by davereinhart** [comment]
> > @kschwabefry ETL changes look good. I was able to test that part and confirm the `daily_travel` values are populating in my local DB. I'm not getting any rows back from `shipping.__uw_priority_queue_v1` view though, even after refreshing the materialized views. I need to dig into that query a little more to see why.
> 
> Wonderful, thanks for testing that, glad to hear that's populating. I have the sqitch changes deployed locally and dumped the prod data yesterday, and `shipping.__uw_priority_queue_v1` is currently pulling up ~139 records in my local db, with symptomatic/exposure as reason selected for testing. The exact number of results in that view would be dependent on that state of the data when you dump from production though, and when you run the query. Not sure if this is the issue, but if you didn't have any recent daily attestation encounters in your local db, that query might not have results, because it is looking for positive attestations within a recent timeframe. I'd expect if you just ingested recent travel attestations, that those would show up in the view though.

That would explain it. I'll refresh my local db and try again. Thanks!

File: schema/deploy/shipping/views.sql [hunk 1]
```diff
@@ -3528,7 +3465,7 @@ create or replace view shipping.__uw_priority_queue_v1 as (
         from uw_enrollments
         -- Filter to participants who enrolled after this date when
         -- baseline test invitations were handled manually
-        where encountered > '2020-12-15'
+        where encountered > '2021-11-15'
```
Comment (davereinhart) [inline schema/deploy/shipping/views.sql:3468]: This change is what is filtering out all of the rows from this view on my local db. If I change it back to the original date, I get 5905 rows but with the new date I get none (local database refreshed yesterday - 11/16).

Comment (kschwabefry) [reply schema/deploy/shipping/views.sql:3468]: This date filter is just limiting the baseline testing invitations, correct? I don't think its expected to limit the other categories - and doesn't seem to in my testing. When I run the priority queue view, I see participants in the queue for  exposure_to_known_positive, symptomatic, and travel testing.

File: lib/seattleflu/id3c/cli/command/offer_uw_testing.py [hunk 2]
```diff
@@ -293,9 +293,9 @@ def testing_type(priority_reason: str) -> TestingType:
         'symptomatic':                  TestingType.SymptomsOrExposure,
         'exposure_to_known_positive':   TestingType.SymptomsOrExposure,
         'gathering_over_10':            TestingType.SymptomsOrExposure,
+        'travel':                       TestingType.SymptomsOrExposure,
         'baseline':                     TestingType.Baseline,
         'surveillance':                 TestingType.Surveillance,
         'surge_testing':                TestingType.ContactTracing,
-        'uw_housing_resident':          TestingType.UwHousingResident,
```
[the remark this answers is not in the document]
Comment (kschwabefry) [reply lib/seattleflu/id3c/cli/command/offer_uw_testing.py:299]: Sure thing, I'll note this as a task for follow up. Thanks!

**Comment by kschwabefry** [comment]
Discussed on slack, going to go ahead and merge/deploy this one tonight.

**Comment by kschwabefry** [comment]
Deployed.

**Pull request merged**